pkg/store/bucket.go

```diff
@@ -257,6 +259,9 @@ type BucketStore struct {
 	// When used with in-memory cache, memory usage should decrease overall, thanks to postings being smaller.
 	enablePostingsCompression   bool
 	postingOffsetsInMemSampling int
+
+	// Enables hits in the Series() response.
```

Comment:
Let's not enable hits (: 
![](https://media1.giphy.com/media/xTiN0r5VQwVbPFyuFW/giphy-downsized-medium.gif)

```suggestion
	// Enables hints in the Series() response.
```